src/main.css

```diff
@@ -1,3 +1,27 @@
 .no-underline {
   text-decoration: none !important;
 }
+
+.col-options {
+  right: 0;
+}
+
+@media screen and (min-width: 200px) and (max-width: 640px) {
+  .col-options {
+    position: relative !important;
+  }
+
+  .col-options.col-3 {
```

Comment:
Definitely not a fan of this. Use the responsive grid classes for this.

Reply:
Yep but I used automatics `col` and to archieve that I should add/remove classes 😟 If you can help me on that do not hesitate 👍

Reply:
I'll have a look at it this weekend